tests/v2/actions/etcdsnapshot/etcdsnapshot.go

```diff
@@ -124,14 +129,19 @@ func RKE2K3SRetentionLimitCheck(client *rancher.Client, clusterName string) erro
 		isS3 = true
 	}
 
-	existingSnapshots, err := etcdsnapshot.GetRKE2K3SSnapshots(client, clusterName)
+	query, err := url.ParseQuery(fmt.Sprintf("labelSelector=%s=%s", etcdsnapshot.SnapshotClusterNameLabel, clusterName))
+	if err != nil {
+		return err
+	}
+
+	snapshotSteveObjList, err := client.Steve.SteveType(etcdsnapshot.SnapshotSteveResourceType).List(query)
```

Comment:
I'm curious if this has been tested for S3? This specifically doesn't yield the exact results as local snapshots, so just calling it out.

Reply:
ack